pkg/cleaner/aws/aws.go

```diff
@@ -105,6 +113,11 @@ func (a *Cleaner) cleanStacks() error {
 
 		a.logger.Log("level", "info", "message", fmt.Sprintf("found that stack %#q should be deleted", *stack.StackName))
 
+		err = a.disableMasterTerminationProtection(*stack.StackName)
```

Comment:
At first I wondered why we disable termination protection twice, but I figured there's one on the EC2 instance and another on the cloudformation stack.

It would be nice to have a log message above, which describe what we are doing here.
```go
"disabling termination protection for EC2 instance"
```

Reply:
done. I have also followed the same pattern and continue deleting stacks even errors is found